examples/battery_storage_parallel.jl

```diff
@@ -77,7 +77,7 @@ println("library loaded")
     end
 
     function constraint(t, x, u, xi)
-    	return( (x[1] <= s_bounds[1][2] )&(x[1] >= s_bounds[1][1]))
+    	return true
```

Comment:
Could it be better to define a default function constraints(t, x, u ,xi) which is defined in the constructor of StochDynProgModel unless explicitly defined by the user?